### ISSUE TITLE
APA article created using rmarkdown::render(), not rmarkdown::draft()

### ISSUE THREAD

**Issue by m-Py** [crsh/papaja#36]
Today I got papaja to run outside of RStudio in Ubuntu 14.04. It was not easy because it took me some time to install a recent version of pandoc and then call it from within R.

In the readme file of this repository you describe to use rmarkdown::draft() in order to create an APA article if you are not using RStudio [ like this: rmarkdown::draft("mymanuscript.Rmd", template = "apa6", package = "papaja") ]. This does not seem to work for me. I get this error message: *The file 'mymanuscript.Rmd' already exists*.

If I just use a different file name I get a weird .rmd document containing this abstract: 

"abstract: In der Rede wird folgende These diskutiert: "Optozentristisch durchseucht sind Philosophie und die Wissenschaft. [...]"

It seems that draft() does not create a .pdf file, but rather tries to create a new file from a template (see http://rmarkdown.rstudio.com/developer_document_templates.html)

To create a papaja APA6 article, I needed to use the rmarkdown function render(), instead of draft(). Using rmarkdown::render(), I could replicate creating your example article as well as articles that contained changes. How to use it:

render("myExample.rmd", output_file="myExample.pdf")

**tl;dr**: If you are using papaja outside of RStudio, it is possible that you can only create an article by calling rmarkdown::render(), not rmarkdown::draft().

**Comment by crsh** [comment]
Hi Martin, thanks for letting me know. I updated to README.
